test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Build.AspNet/InstanceAnnotations/AnnotationEdmModel.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using Microsoft.AspNet.OData.Builder;
+using Microsoft.OData.Edm;
+using Microsoft.Test.E2E.AspNet.OData.Common.Execution;
+using NS;
+
+namespace Microsoft.Test.E2E.AspNet.OData.InstanceAnnotations
+{
+    internal class AnnotationEdmModel
+    {
+        public static IEdmModel GetExplicitModel(WebRouteConfiguration configuration)
+        {
+            ODataModelBuilder builder = new ODataModelBuilder();
+            var employee = builder.EntityType<Employee>();
+            employee.HasKey(c => c.ID);
+            employee.Property(c => c.Name);
+            employee.CollectionProperty<Skill>(c => c.SkillSet);
+            employee.EnumProperty<Gender>(c => c.Gender);
+            employee.EnumProperty<AccessLevel>(c => c.AccessLevel);
+            employee.ComplexProperty<FavoriteSports>(c => c.FavoriteSports);
+            employee.Namespace = "NS";
```

Comment:
I can't see you set the instance annotation configuration in "ExplictiModel" scenario

Reply:
We Removed HasInstanceannotation, instead relaying on any object of type IODataInstanceAnnotationContainer to be for instance annotation

Reply:
From Sreejith: We Removed HasInstanceannotation, instead relaying on any object of type IODataInstanceAnnotationContainer to be for instance annotation

---
In reply to: [489015943](https://github.com/OData/WebApi/pull/2219#discussion_r489015943) [](ancestors = 489015943)

Reply:
In this case, we 'd test the "Explicit" scenario. In you AnnotationTest, you only test the "Convention" scenario.

---
In reply to: [489816435](https://github.com/OData/WebApi/pull/2219#discussion_r489816435) [](ancestors = 489816435)